src/garage/experiment/local_runner.py

```diff
@@ -165,39 +178,47 @@ def restore(self, from_dir, from_epoch='last'):
                 Not applicable when snapshot_mode='last'.
 
         Returns:
-            A SimpleNamespace for train()'s arguments.
+            types.SimpleNamespace: A SimpleNamespace for train()'s arguments.
 
         """
         saved = self._snapshotter.load(from_dir, from_epoch)
 
         self._setup_args = saved['setup_args']
-        self.train_args = saved['train_args']
+        self._train_args = saved['train_args']
+        self._stats = saved['stats']
+
+        set_seed(self._setup_args.seed)
 
         self.setup(env=saved['env'],
                    algo=saved['algo'],
                    sampler_cls=self._setup_args.sampler_cls,
                    sampler_args=self._setup_args.sampler_args)
 
-        n_epochs = self.train_args.n_epochs
-        last_epoch = saved['last_epoch']
-        n_epoch_cycles = self.train_args.n_epoch_cycles
-        batch_size = self.train_args.batch_size
-        store_paths = self.train_args.store_paths
-        pause_for_plot = self.train_args.pause_for_plot
+        n_epochs = self._train_args.n_epochs
+        last_epoch = self._stats.total_epoch
+        last_itr = self._stats.total_itr
+        total_env_steps = self._stats.total_env_steps
+        n_epoch_cycles = self._train_args.n_epoch_cycles
+        batch_size = self._train_args.batch_size
+        store_paths = self._train_args.store_paths
+        pause_for_plot = self._train_args.pause_for_plot
 
         fmt = '{:<20} {:<15}'
         logger.log('Restore from snapshot saved in %s' %
                    self._snapshotter.snapshot_dir)
-        logger.log(fmt.format('Train Args', 'Value'))
+        logger.log(fmt.format('-- Train Args --', '-- Value --'))
         logger.log(fmt.format('n_epochs', n_epochs))
         logger.log(fmt.format('last_epoch', last_epoch))
         logger.log(fmt.format('n_epoch_cycles', n_epoch_cycles))
         logger.log(fmt.format('batch_size', batch_size))
         logger.log(fmt.format('store_paths', store_paths))
         logger.log(fmt.format('pause_for_plot', pause_for_plot))
+        logger.log(fmt.format('-- Stats --', '-- Value --'))
+        logger.log(fmt.format('last_itr', last_itr))
+        logger.log(fmt.format('total_env_steps', total_env_steps))
```

Comment:
this is only useful to us if it's logged to `tabular`, because `tabular` logs are sent to TensorBoardOutput (where we can use this key for the x-axis). 

there is a separate issue to unify `tabular` and `log` in dowel (by adding either logger.logkv(key, value) or logger.log(value, key=None)` options, but i think it can be out-of-scope for this PR